examples/combined_ODNP.py

```diff
@@ -28,6 +31,7 @@
 config_dict["date"] = date
 config_dict["odnp_counter"] += 1
 filename = f"{config_dict['date']}_{config_dict['chemical']}_{config_dict['type']}_{config_dict['odnp_counter']}"
+filename_out = filename + ".h5"
```

Comment:
I don't understand. What is the point of breaking this across two lines of introducing a new variable?